configure.ac

```
@@ -64,7 +64,10 @@ IFS=$ac_IFS
 PHP_MAJOR_VERSION=[$]1
 PHP_MINOR_VERSION=[$]2
 PHP_RELEASE_VERSION=[$]3
-PHP_EXTRA_VERSION=[$]4
+dnl Allow overriding PHP_EXTRA_VERSION through the homonymous env var
+if test -z "$PHP_EXTRA_VERSION"; then
+  PHP_EXTRA_VERSION=[$]4
+fi
```

```suggestion
AC_ARG_VAR([PHP_EXTRA_VERSION],[Extra PHP version label suffix, e.g. '-dev', '-rc1', '-acme'])
AS_IF([test -z "$PHP_EXTRA_VERSION"],[PHP_EXTRA_VERSION=[$]4])
```

This will add the `PHP_EXTRA_VERSION` to `./configure --help` output so users know which variables are available.